 selfdrive/car/toyota/carstate.py | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/selfdrive/car/toyota/carstate.py b/selfdrive/car/toyota/carstate.py
index bb3b3f05..80ec37ea 100644
--- a/selfdrive/car/toyota/carstate.py
+++ b/selfdrive/car/toyota/carstate.py
@@ -170,3 +170,12 @@ class CarState(object):
       self.generic_toggle = cp.vl["AUTOPARK_STATUS"]['STATE'] != 0
     else:
       self.generic_toggle = bool(cp.vl["LIGHT_STALK"]['AUTO_HIGH_BEAM'])
+
+    if self.generic_toggle and self.main_on:
+      self.pcm_acc_active = True
+      if self.standstill:
+        # self.pcm_acc_active = False
+        self.pcm_acc_status = 7
+      else:
+        # self.pcm_acc_active = True
+        self.pcm_acc_status = 1
